LadderManager.cpp

```diff
@@ -897,6 +913,12 @@ void LadderManager::LoadAgents()
 			if (val.HasMember("Path") && val["Path"].IsString())
 			{
 				NewBot.Path = val["Path"].GetString();
+				if (NewBot.Type != DefaultBot && !sc2::DoesFileExist(NewBot.Path))
+				{
+					std::cerr << "Unable to parse bot " << NewBot.Name << std::endl;
+					std::cerr << "Is the path " << NewBot.Path << "correct?"<<std::endl;
+					continue;
```

Comment:
Are you sure that we can continue with invalid config?

Reply:
Thats how it works at the moment, we just don't add this bot to the array.  Not sure if its worth changing that at the moment

Reply:
Ok, I've missed that)